invokeai/frontend/web/src/features/nodes/components/flow/nodes/Invocation/fields/InputField.tsx

```diff
@@ -46,6 +48,14 @@ const InputField = ({ nodeId, fieldName }: Props) => {
     return false;
   }, [fieldTemplate, isConnected, doesFieldHaveValue]);
 
+  const handleMouseOver = useCallback(() => {
+    setIsHovered(true);
+  }, []);
+
+  const handleMouseOut = useCallback(() => {
+    setIsHovered(false);
+  }, []);
+
```

Comment:
Let's name these according to the events they represent (e.g. onMouseEnter & onMouseLeave)